frontend/package.json

```diff
@@ -24,7 +24,8 @@
     "test-suite": "ts-node -O '{\"module\":\"commonjs\"}' ./node_modules/.bin/protractor integration-tests/protractor.conf.ts",
     "debug-test-suite": "TS_NODE_COMPILER_OPTIONS='{\"module\":\"commonjs\"}' node -r ts-node/register --inspect-brk ./node_modules/.bin/protractor integration-tests/protractor.conf.ts",
     "analyze": "NODE_ENV=production NODE_OPTIONS=--max-old-space-size=4096 ts-node -O '{\"module\":\"commonjs\"}' ./node_modules/.bin/webpack --mode=production --profile --json | awk '{if(NR>2)print}' > public/dist/stats.json && ts-node -O '{\"module\":\"commonjs\"}' ./node_modules/.bin/webpack-bundle-analyzer --mode static -r public/dist/report.html public/dist/stats.json",
-    "plugin-stats": "node ./plugin-stats.js"
+    "plugin-stats": "node ./plugin-stats.js",
+    "prettier": "prettier --write '**/*.{js,jsx,ts,tsx}'"
```

Comment:
Do we want to also format `json`? Doing so currently affects 23 files.

Reply:
We should.
Can we update eslint to run against json files as well so it's enforced by our ci?
Then we fix them all up and can include json files w/ prettier here.

Reply:
I've added eslint-plugin-json for json linting and created `json.js` in our `eslint-plugin-console' package.